metricbeat/module/elasticsearch/ml_job/_meta/test/xpack.job.stats.json

```diff
@@ -0,0 +1,62 @@
+{
+    "_index": ".monitoring-es-6-2018.05.09",
+    "_type": "doc",
+    "_id": "1QWfQ2MBM5SQsInIyGuQ",
+    "_score": 2.6701682,
+    "_source": {
+        "cluster_uuid": "EFdYsDSoSZmU7fXE2wvJ0g",
+        "timestamp": "2018-05-09T06:38:55.058Z",
+        "interval_ms": 10000,
+        "type": "job_stats",
+        "source_node": {
+            "uuid": "q7ya3ngBTS2cxzYi0duQdA",
+            "host": "127.0.0.1",
+            "transport_address": "127.0.0.1:9300",
+            "ip": "127.0.0.1",
+            "name": "q7ya3ng",
+            "timestamp": "2018-05-09T06:38:54.931Z"
+        },
+        "job_stats": {
+            "job_id": "test",
+            "data_counts": {
+                "job_id": "test",
+                "processed_record_count": 0,
+                "processed_field_count": 0,
+                "input_bytes": 0,
+                "input_field_count": 0,
+                "invalid_date_count": 0,
+                "missing_field_count": 0,
+                "out_of_order_timestamp_count": 0,
+                "empty_bucket_count": 0,
+                "sparse_bucket_count": 0,
+                "bucket_count": 0,
+                "input_record_count": 0
+            },
+            "model_size_stats": {
+                "job_id": "test",
+                "result_type": "model_size_stats",
+                "model_bytes": 0,
+                "total_by_field_count": 0,
+                "total_over_field_count": 0,
+                "total_partition_field_count": 0,
+                "bucket_allocation_failures_count": 0,
+                "memory_status": "ok",
+                "log_time": 1525847933184
```

Comment:
Are we going to collect more metrics?

Reply:
Yes. This PR mainly creates the foundation.